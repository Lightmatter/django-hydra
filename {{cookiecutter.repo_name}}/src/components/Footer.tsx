import { Paper, Typography } from '@material-ui/core';
import PropTypes from 'prop-types';

import Link from '@material-ui/core/Link';

const thisyear = new Date().getFullYear();
const Footer = ({ className }) => {
  return (
    <Paper className={className} elevation={0}>
      <Typography variant="body2" color="textSecondary" align="center">
        {'Copyright Â© '}
        <Link color="inherit" href="https://www.lightmatter.com/">
          Lightmatter
        </Link>{' '}
        {thisyear}
        {'.'}
      </Typography>
    </Paper>
  );
};

Footer.propTypes = {
  className: PropTypes.string,
};

Footer.defaultProps = {
  className: '',
};

export default Footer;
