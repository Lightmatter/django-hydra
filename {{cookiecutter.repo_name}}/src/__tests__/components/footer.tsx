import React from 'react';
import { createMount } from '@material-ui/core/test-utils';
import { mount } from 'enzyme';
import Footer from 'components/Footer';

describe('Footer', () => {
  it('renders footer unchanged', () => {
    const tree = mount(<footer />);
    expect(tree).toMatchSnapshot();
  });
});
