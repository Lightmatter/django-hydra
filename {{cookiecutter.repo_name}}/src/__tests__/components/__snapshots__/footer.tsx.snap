// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders footer unchanged 1`] = `<footer />`;
